### PR TITLE
test refactor to use helper for duplicates in light client committees

### PR DESCRIPTION
**Pull request by ralexstokes**
patch to scan seed space until we find (or do not find) duplicates in the light client sync committees, regardless of config settings

### CODE REVIEW

**Comment by ralexstokes** [comment]
probably going to close this; see https://github.com/ethereum/eth2.0-specs/pull/2181#issuecomment-764139641

File: tests/core/pyspec/eth2spec/test/lightclient_patch/block_processing/test_process_sync_committee.py [hunk 1]
```diff
@@ -74,10 +93,9 @@ def compute_sync_committee_participant_reward(spec, state, participant_index, ac
 
 
 @with_all_phases_except([PHASE0, PHASE1])
-@with_configs([MINIMAL], reason="to create nonduplicate committee")
```
Comment (hwwhww) [inline tests/core/pyspec/eth2spec/test/lightclient_patch/block_processing/test_process_sync_committee.py:77]: I think we should to keep this decorator to reduce the iteration times. (CI running out of time now)

File: tests/core/pyspec/eth2spec/test/lightclient_patch/block_processing/test_process_sync_committee.py [hunk 2]
```diff
@@ -123,10 +141,9 @@ def test_sync_committee_rewards_nonduplicate_committee(spec, state):
 
 
 @with_all_phases_except([PHASE0, PHASE1])
-@with_configs([MAINNET], reason="to create duplicate committee")
```
Comment (hwwhww) [inline tests/core/pyspec/eth2spec/test/lightclient_patch/block_processing/test_process_sync_committee.py:126]: ditto

File: tests/core/pyspec/eth2spec/test/lightclient_patch/block_processing/test_process_sync_committee.py [hunk 3]
```diff
@@ -123,10 +141,9 @@ def test_sync_committee_rewards_nonduplicate_committee(spec, state):
...
 @spec_state_test
```
Comment (hwwhww) [inline tests/core/pyspec/eth2spec/test/lightclient_patch/block_processing/test_process_sync_committee.py:144]:
```suggestion
@with_configs([MAINNET], reason="to create duplicate committee")
@spec_state_test
```

File: tests/core/pyspec/eth2spec/test/lightclient_patch/block_processing/test_process_sync_committee.py [hunk 4]
```diff
@@ -74,10 +93,9 @@ def compute_sync_committee_participant_reward(spec, state, participant_index, ac
 
 
 @with_all_phases_except([PHASE0, PHASE1])
-@with_configs([MINIMAL], reason="to create nonduplicate committee")
 @spec_state_test
```
Comment (hwwhww) [inline tests/core/pyspec/eth2spec/test/lightclient_patch/block_processing/test_process_sync_committee.py:96]:
```suggestion
@with_configs([MINIMAL], reason="to create nonduplicate committee")
@spec_state_test
```

**Review by hwwhww — COMMENTED** [review]
Thank @ralexstokes! And thank @djrtwo for the comment.
I like this approach! 👍

File: tests/core/pyspec/eth2spec/test/lightclient_patch/block_processing/test_process_sync_committee.py (hunk 1 above)
Comment (ralexstokes) [reply tests/core/pyspec/eth2spec/test/lightclient_patch/block_processing/test_process_sync_committee.py:77]: yes agreed

**Pull request merged**